src/main/java/alfio/repository/TicketCategoryRepository.java

```diff
@@ -75,7 +75,7 @@
     @Query("select * from ticket_category_with_currency where event_id = :eventId  and tc_status = 'ACTIVE' order by inception asc, expiration asc, id asc")
     List<TicketCategory> findAllTicketCategories(@Bind("eventId") int eventId);
     
-    @Query("select * from ticket_category_with_currency where event_id = :eventId")
+    @Query("select * from ticket_category_with_currency where event_id = :eventId and tc_status = 'ACTIVE'")
```

Comment:
why the additional check on the status? :0

Reply:
Removed the method entirely, as it was overlapping with findAllTicketCategories.
BTW the status is needed because until now we had the column on the DB, but we didn't use it :)